crates/iota-indexer/src/ingestion/common/persist.rs

```diff
@@ -33,7 +33,7 @@ pub trait Writer<T: Send + Sync + 'static>: Send + Sync {
     async fn persist(&self, batch: Vec<T>) -> IndexerResult<()>;
 
     /// Reads high watermark of the table DB.
-    async fn get_watermark_hi(&self) -> IndexerResult<Option<CommitterWatermark>>;
+    async fn get_watermark_hi(&self) -> IndexerResult<Option<u64>>;
```

Comment:
nit: Instead of u64 we could use the alias type `CheckpointSequenceNumber`

Reply:
Done in 81a1d96f00c5b3b140854730315c7c6a9b091cd3